src/app/assets/json/quiz.json

```diff
@@ -8,19 +8,23 @@
                 "questionType": 1,
                 "options": [{
                         "name": "<javascript>",
-                        "isAnswer": false
+                        "isAnswer": false,
+                        "isSelected": false
```

Comment:
Hi @Akshayakt 
Approved json structure should not be edited. And please move this logic to component only.